lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -46,10 +46,20 @@ def save_to_file(filename, content)
             "cat << EOF > #{filename}\n#{content}EOF"
           end
 
-          def indent(count)
+          def indent(count, skip1: false)
             return unless block_given? && (text = yield.to_s)
-            prefix = " " * count
-            prefix + text.gsub(/\n/, "\n#{prefix}")
+            prefix = ' ' * count
+            result = []
+            line_no = 0
+            text.each_line do |line|
```

Comment:
Why not `with_index`?

```ruby
text.each_line.with_index do |line, line_no|
  # ...
end
```

Reply:
Oh nice, I could not figure out this one :-)